frontend/packages/dev-console/src/components/helm/HelmReleaseRow.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { Status } from '@console/shared';
+import { TableRow, TableData } from '@console/internal/components/factory';
+import { Timestamp } from '@console/internal/components/utils';
+import { Link } from 'react-router-dom';
+import { HelmRelease } from './helm-types';
+import { tableColumnClasses } from './HelmReleaseHeader';
+
+interface HelmReleaseRowProps {
+  obj: HelmRelease;
+  index: number;
+  key?: string;
+  style: object;
+}
+
+const HelmReleaseRow: React.FC<HelmReleaseRowProps> = ({ obj, index, key, style }) => {
+  return (
+    <TableRow id={obj.name} index={index} trKey={key} style={style}>
+      <TableData className={tableColumnClasses.name}>
+        <Link
+          to={`/helm-releases/ns/${obj.namespace}/release/${obj.name}`}
+          title={obj.name}
+          className="co-resource-item__resource-name"
+          data-test-id={obj.name}
```

Comment:
Why this dynamic test I'd here? I don't see it being used.

Reply:
Removed.